tests/expectations/metrics/column_map_metrics/test_unexpected_indices_and_query.py

```diff
@@ -270,8 +270,9 @@ def test_sa_unexpected_index_query_metric_with_id_pk(
     )
     for (key, val) in results.items():
         assert (
-            val
-            == "SELECT animals, pk_1 \nFROM test \nWHERE animals IS NOT NULL AND (animals NOT IN ('cat', 'fish', 'dog'));"
+            "SELECT pk_1, animals \n"
+            "FROM test \n"
+            "WHERE animals IS NOT NULL AND (animals NOT IN ('cat', 'fish', 'dog'));"
         )
```

Comment:
Looks like this will always be true? Does this need to be compared to val like in the previous version?

Reply:
@anthonyburdi yes thank you for spotting this. Yes, this will always be `True`. The reason why we had to compare to other versions in the Pandas PR is because the method had to take into account default indices for Pandas Dataframes (which SQL doesn't have), which depended on a `groupby()` stage that had multiple potential outputs. Therefore, the list of columns is always going to be in the same order.

Reply:
But shouldn't we still compare to the query (`val` I think here) that we are getting in the results to the expected query? To make sure the output query is correct? The proposed change means we are just asserting that it is a non-empty string.

Reply:
oh nooooo. You're right. Let me fix this real quick :)